tensorflow_addons/image/transform_test.py

```diff
@@ -141,6 +147,156 @@ def test_transform_eager(self):
+        for dtype in _DTYPES:
+            with self.cached_session():
+                image = array_ops.reshape(
+                    math_ops.cast(math_ops.range(36), dtype), (6, 6))
+                image_rep = array_ops.tile(image[None, :, :, None],
+                                           [3, 1, 1, 1])
+                angles = constant_op.constant([0.0, np.pi / 4.0, np.pi / 2.0],
+                                              dtypes.float32)
+                image_rotated = transform_ops.rotate(image_rep, angles)
+                # yapf: disable
+                self.assertAllEqual(
+                    image_rotated[:, :, :, 0],
+                    [[[0, 1, 2, 3, 4, 5],
+                      [6, 7, 8, 9, 10, 11],
+                      [12, 13, 14, 15, 16, 17],
+                      [18, 19, 20, 21, 22, 23],
+                      [24, 25, 26, 27, 28, 29],
+                      [30, 31, 32, 33, 34, 35]],
+                     [[0, 3, 4, 11, 17, 0],
+                      [2, 3, 9, 16, 23, 23],
+                      [1, 8, 15, 21, 22, 29],
+                      [6, 13, 20, 21, 27, 34],
+                      [12, 18, 19, 26, 33, 33],
+                      [0, 18, 24, 31, 32, 0]],
+                     [[5, 11, 17, 23, 29, 35],
+                      [4, 10, 16, 22, 28, 34],
+                      [3, 9, 15, 21, 27, 33],
+                      [2, 8, 14, 20, 26, 32],
+                      [1, 7, 13, 19, 25, 31],
+                      [0, 6, 12, 18, 24, 30]]])
+                # yapf: enable
+
+    @test_utils.run_in_graph_and_eager_modes
+    def test_rotate_odd(self):
+        for dtype in _DTYPES:
+            with self.cached_session():
+                image = array_ops.reshape(
+                    math_ops.cast(math_ops.range(25), dtype), (5, 5))
+                image_rep = array_ops.tile(image[None, :, :, None],
+                                           [3, 1, 1, 1])
+                angles = constant_op.constant([np.pi / 4.0, 1.0, -np.pi / 2.0],
+                                              dtypes.float32)
+                image_rotated = transform_ops.rotate(image_rep, angles)
+                # yapf: disable
+                self.assertAllEqual(
+                    image_rotated[:, :, :, 0],
+                    [[[0, 3, 8, 9, 0],
+                      [1, 7, 8, 13, 19],
+                      [6, 6, 12, 18, 18],
+                      [5, 11, 16, 17, 23],
+                      [0, 15, 16, 21, 0]],
+                     [[0, 3, 9, 14, 0],
+                      [2, 7, 8, 13, 19],
+                      [1, 6, 12, 18, 23],
+                      [5, 11, 16, 17, 22],
+                      [0, 10, 15, 21, 0]],
+                     [[20, 15, 10, 5, 0],
+                      [21, 16, 11, 6, 1],
+                      [22, 17, 12, 7, 2],
+                      [23, 18, 13, 8, 3],
+                      [24, 19, 14, 9, 4]]])
+                # yapf: enable
+
+    def test_compose_rotate(self):
```

Comment:
Could you add `test_utils.run_in_graph_and_eager_modes` decorator for each test methods?

Reply:
Done